stock_market_simulator/src/pages/Stocks.jsx

```diff
@@ -81,7 +81,8 @@ function Stocks(props) {
             <div className="stock-buy-left">
               <h1>Buy {stock.symbol}</h1>
               <h2>Current price: &euro; {stock.price}</h2>
```

Comment:
Is het nu ook enigszins gek dat de huidige prijs en de totaal prijs niet overeen komen toch?
Current price: € 1002.14
Total: € 1004.645